capsul/qt_gui/widgets/pipeline_developper_view.py

```diff
@@ -3136,7 +3106,7 @@ def open_node_menu(self, node_name, process):
             # allow to select switch value from the menu
             submenu = menu.addMenu('Switch value')
             agroup = QtGui.QActionGroup(submenu)
-            values = node.trait('switch').trait_type.values
+            values = node.field('switch').type.__args__[0].__args__
```

Comment:
There are two functions `subtypes(type)` and `field_subtypes(field)` in `soma.controller.field`. To date they are just using `__args__` but I already encountered some differences in some internal attributes for some similar types definition (such as `List[...]` and `list[...]`). Using this function will allow us to deal with these difference and to manage backward compatibility.

Reply:
OK, good to know. I'm changing it.